resources/views/site/blocks/custom_banner.blade.php

```diff
@@ -0,0 +1,59 @@
+@php 
+
+    $bg_type = $block->input('background_type');
+    $btn_type = $block->input('button_type');
+
+    $heading = $block->input('heading');
+    $body = $block->input('body');
+
+@endphp
+
+<div class="m-custom_banner-block">
+    <div class="content-wrapper">
+        <div class="background-wrapper">
+            @if($bg_type == 'background_color')
+                <div class="background_color" style="background-color: {!! $block->input('bgcolor') !!}"></div>
```

Comment:
Add a check if `$block->input('bgcolor')` is not set. You can add it around the entire "style" attribute:

```
{{ isset($block->input('bgcolor')) ? 'style="background-color: ' . $block->input('bgcolor') . '"' : '' }}